src/test/java/htsjdk/samtools/SAMReadGroupRecordTest.java

```diff
@@ -0,0 +1,42 @@
+/*
+ * The MIT License
+ *
+ * Copyright (c) 2009 The Broad Institute
```

Comment:
Haha! (Copied an existing test file from the same package as the starting point and the comment was folded in the editor off the bat! I can update if you want)